db/migrate/20161130153206_add_last_updated_at_to_content_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastUpdatedAtToContentItems < ActiveRecord::Migration[5.0]
+  def change
+    add_column :content_items, :last_updated_at, :datetime
```

Comment:
@ikennaokpala I don't get what you mean by `relative to the context`. You prefer to stick to `last_updated_at`. What is your preference? You don't have to push an update if you don't agree.